Marlin/src/pins/lpc1769/pins_BTT_SKR_E3_TURBO.h

```diff
@@ -203,6 +203,7 @@
 
 #if ENABLED(DWIN_CREALITY_LCD)
   #error "DWIN_CREALITY_LCD requires a custom cable with TX = P0_15, RX = P0_16. Comment out this line to continue."
+  //Make custom cable: https://github.com/bigtreetech/BIGTREETECH-SKR-E3-Turbo/issues/40#issuecomment-866369883
```

Comment:
Instead of adding a link, just add the ascii art directly like we've done for other boards:
https://github.com/MarlinFirmware/Marlin/blob/2c49283e97f4bb9cac9577af7c38bcf5e0e06826/Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_common.h#L137-L149